src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs

```diff
@@ -126,9 +126,50 @@
                    (:workflows props))}])))})
 
 
+(react/defc AbortSubmission
```

Comment:
By "this" I mean the AbortSubmission component.  It's not just an abort button, it's the status label and the abort button.